File: fizzbuzz.py

def fizzbuzz_program():
    # Loops through numbers 1 - 100
    for i in range(1,101):
        #Checks if i (number) is divisible by 3 and 5 first
        if i % 5 == 0 and i % 3 == 0:
            print ("FizzBuzz")
        elif i % 5 == 0:
            print ("Buzz")
        elif i % 3 == 0:
            print ("Fizz")
        else:
            print ("Number is not divisible by 3 or 5")

def main():
    # Calls the fizzbuzz_program
    fizzbuzz_program()

# Calls the main function
if __name__ == '__main__':
    main()

File: fizzbuzz.java
// import scanner
import java.util.Scanner;

public class FizzBuzz
{
    public static void main(String[] args){
        int number;
        Scanner in;

        // Get number by user
        in = new Scanner(System.in);
        System.out.println("Enter a number between 1 and 100: ");
        number = in.nextInt();
        

        
    //Determine whether to print Fizz, Buzz or FizzBuzz
    if (number % 3 == 0 & number % 5 == 0) {
        System.out.println("FizzBuzz");}
    else if (number % 3 == 0) {
        System.out.println("Fizz");}
    else if (number % 5 == 0) {
        System.out.println("Buzz");}
    else {
        System.out.println("Number is not divisible by 3 or 5");
    }
    
}
}